### PR TITLE
Update pss.h

### PR DESCRIPTION
**Pull request by KuromeSan**
Add some functions

### CODE REVIEW

File: include/psp2/pss.h [hunk 1]
```diff
@@ -12,7 +12,17 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+  
+typedef struct PSM_handle {
```
Comment (xerpi) [inline include/psp2/pss.h]: To be consistent, I'd change the name to something like `ScePssHandle`, or maybe `ScePssCryptoHandle` if this handle is only used for PSS crypto functions.

File: include/psp2/pss.h [hunk 2]
```diff
@@ -12,7 +12,17 @@
...
+uint32_t unk0;
```
Comment (xerpi) [inline include/psp2/pss.h]: Also indent the struct members (tab).

**Comment by KuromeSan** [comment]
that good?

File: include/psp2/pss.h [hunk 3]
```diff
@@ -12,7 +12,16 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-
+  
+typedef struct ScePssHandle {
```
Comment (devnoname120) [inline include/psp2/pss.h]: `ScePssHandle` is unnecessary, remove it.

File: include/psp2/pss.h [hunk 4]
```diff
@@ -12,7 +12,16 @@
...
+  uint32_t unk0;
+  uint32_t unk1;
+  uint32_t filesz;
+  uint32_t unk3;
+} ScePssCryptoHandle;
+
+int pss_crypto_open_p(ScePssCryptoHandle *handle, char *path);
+char *pss_crypto_read_p(ScePssCryptoHandle *handle, int unk);
```
Comment (devnoname120) [inline include/psp2/pss.h]: Can you provide a known good value for `unk`?

Check other source files to see how you can document a function.

Comment (KuromeSan) [reply include/psp2/pss.h]: It's a pointer to an int appears to always be 0x1 from what I've seen ?

**Comment by xerpi** [comment]
Looks good to me.

**Pull request merged**